web/filament-js/wasmloader.js

```diff
@@ -82,3 +82,23 @@ Filament.postRun = function() {
         Filament.onReady();
     }
 };
+
+/// fetch ::function:: Downloads assets and invokes a callback when done.
+/// assets ::argument:: Array of strings containing URL's of required assets.
+/// onready ::argument:: callback that gets invoked after all assets have been downloaded.
+Filament.fetch = function(assets, onDone) {
```

Comment:
This function will be used in the new tutorial, it works similar to `init` but make it easy for clients to download additional assets after the app has been loaded.